### ISSUE TITLE
CMake-built flameshot did not install all data files needed

### ISSUE THREAD

**Issue by hosiet** [flameshot-org/flameshot#815]
**Flameshot version**
git HEAD

**Describe the bug**
The installation of flameshot on a UNIX-like system is not ONLY a `/usr/bin/flameshot`. There are much more
data files that needs to be installed as well.

Currently a `make install` will only install `/usr/bin/flameshot`. We need to add instructions in CMake buildsystem
to also install other data files.

A full list of data files:

```
/usr/bin/flameshot
/usr/share/applications/flameshot.desktop
/usr/share/bash-completion/completions/flameshot
/usr/share/dbus-1/interfaces/org.dharkael.Flameshot.xml
/usr/share/dbus-1/services/org.dharkael.Flameshot.service
/usr/share/flameshot/translations/Internationalization_ca.qm
/usr/share/flameshot/translations/Internationalization_de_DE.qm
/usr/share/flameshot/translations/Internationalization_es.qm
/usr/share/flameshot/translations/Internationalization_fr.qm
/usr/share/flameshot/translations/Internationalization_ja.qm
/usr/share/flameshot/translations/Internationalization_ka.qm
/usr/share/flameshot/translations/Internationalization_pl.qm
/usr/share/flameshot/translations/Internationalization_pt_br.qm
/usr/share/flameshot/translations/Internationalization_ru.qm
/usr/share/flameshot/translations/Internationalization_sk.qm
/usr/share/flameshot/translations/Internationalization_sr.qm
/usr/share/flameshot/translations/Internationalization_tr.qm
/usr/share/flameshot/translations/Internationalization_uk.qm
/usr/share/flameshot/translations/Internationalization_zh_CN.qm
/usr/share/flameshot/translations/Internationalization_zh_TW.qm
/usr/share/icons/hicolor/128x128/apps/flameshot.png
/usr/share/icons/hicolor/48x48/apps/flameshot.png
/usr/share/icons/hicolor/scalable/apps/flameshot.svg
/usr/share/man/man1/flameshot.1.gz
/usr/share/metainfo/flameshot.appdata.xml
```

**Comment by borgmanJeremy** [comment]
Working on this now. Do you know what the "lintian" in this path is for: /usr/share/lintian/overrides/flameshot

**Comment by hosiet** [comment]
Oh please ignore the lintian override file, that is Debian-specific.

**Comment by borgmanJeremy** [comment]
What about the man file? Not seeing that in the source tree.

**Comment by hosiet** [comment]
Ah sorry this file is also Debian-specific. You can find a copy at https://sources.debian.org/src/flameshot/0.6.0+git20191001-2/debian/flameshot.1/ .

If you are willing to add this file in the upstream repo, please feel free to do so.

**Comment by borgmanJeremy** [comment]
Checkout #819 I just realized I forgot the man page, I'll add that tonight and fix any other comments you have.

**Comment by borgmanJeremy** [comment]
Merged #819
